templates/reactor-config-secret.yml.j2

```diff
@@ -14,6 +14,11 @@ clusters:
 
 {% if osbs_odcs_enabled %}
 odcs:
+    {% if osbs_odcs_api_url %}
+        api_url: {{ osbs_odcs_api_url }}
+        auth:
+        ssl_certs_dir: {{ osbs_odcs_auth_ssl_certs_dir }}
```

Comment:
This needs to be indented:
```
auth:
  ssl_certs_dir: {{ osbs_odcs_auth_ssl_certs_dir }}
```

Otherwise `auth` and `ssl_certs_dir` are siblings.